docker-compose.yml

```diff
@@ -1,4 +1,35 @@
 services:
+  nginx:
+    build:
+      context: ./
+      dockerfile: ./docker/Dockerfile.nginx
+    cap_add:
+      - NET_BIND_SERVICE
+    ports:
+      - "80:80"
+      - "443:443"   # For the frontend
+      - "8443:8443" # For the jsonrpc
+    volumes:
+      - ./nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+      - ./nginx/ssl:/etc/nginx/ssl:ro
+      - ./nginx/rpc-blocker.lua:/etc/nginx/rpc-blocker.lua:ro
+    depends_on:
+      - frontend
+      - jsonrpc
+    healthcheck:
+      test: ["CMD", "nginx", "-t"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+    restart: always
+    security_opt:
+      - "no-new-privileges=true"
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "3"
```

Comment:
this service needs to be off by default, so it doesn't interfere with local installations

Reply:
I've turned it off by default, use `docker-compose --profile studio up` to start it explicitly